pycsw/core/util.py

```diff
@@ -33,11 +33,15 @@
 import time
 import datetime
 import logging
-import urllib2
+try:
+    from urllib2 import Request, urlopen
+except:
+    from urllib.request import Request, urlopen
 from shapely.wkt import loads
 from owslib.util import http_post
 from pycsw.core.etree import etree
 
+from six import text_type
```

Comment:
> I am ok with adding six as a dependency.

That is better.  If one day `OWSLib` drops `six` than `pycsw` would be broken.  (Explicit is better than implicit.)

> Copying the six.py file in our code base would need some licensing review etc.

I believe that would be OK, but `six` is a lightweight dependency.